src/game/gui/text_render.c

```diff
@@ -8,10 +8,13 @@
 
 void text_defaults(text_settings *settings) {
```

Comment:
I'm not sure if the color indexes should be set here. Do they apply for all scenes ?